packages/components/grid/src/stories/basics.stories.ts

```diff
@@ -71,6 +80,10 @@ export const ColumnGroups: Story = {
 
 export const EllipsizeText: Story = {
```

Comment:
When the text is ellipsized and there is a tooltip visible, when we hover over it, the tooltip should be also visible on focus when using keyboard, so we should be able to focus ellipsized cells.